build/generate.mk

```diff
@@ -116,7 +116,9 @@ generate-go: generate-mockgen-installed ## Generates go with 'go generate' (Mock
 generate-bicep-types: generate-node-installed ## Generate Bicep extensibility types
 	@echo "$(ARROW) Generating Bicep extensibility types from OpenAPI specs..."
 	@echo "$(ARROW) Build autorest.bicep..."
-	cd hack/bicep-types-radius/src/autorest.bicep; \
+	cd bicep-types/src/bicep-types; \
+	npm ci && npm run build; \
```

Comment:
Might be easier to use `--prefix` instead of changing directories? https://stackoverflow.com/questions/37078968/how-to-specify-the-path-of-package-json-to-npm

Reply:
Then you don't have to do math about home many `..`s you need.